src/screens/SettingsScreen/SettingsScreen.tsx

```diff
@@ -1,26 +1,82 @@
 import React from 'react';
-import { Button } from 'react-native';
+import { Alert, TouchableWithoutFeedback } from 'react-native';
 
-import { useDispatch } from 'react-redux';
+import { useNavigation } from '@react-navigation/native';
+import { useTranslation } from 'react-i18next';
+import { ListItem } from 'react-native-elements';
+import { useDispatch, useSelector } from 'react-redux';
 
 import { NavigationData } from '~/navigation/AppNavigator/SettingsNavigator';
 import { logout } from '~/redux/auth';
+import { RootState } from '~/redux/rootReducer';
+import Colors from '~/theme/Colors';
 
 import { StyledView } from './SettingsScreen.styled';
 
 type Props = NavigationData<'Settings'>;
 
 const SettingsScreen: React.FC<Props> = () => {
   const dispatch = useDispatch();
+  const { username, userImage } = useSelector((state: RootState) => state.auth);
+  const { navigate } = useNavigation();
+  const { t } = useTranslation();
 
   return (
     <StyledView>
-      <Button
-        title="Log out"
+      <TouchableWithoutFeedback onPress={() => navigate('UserSettings')}>
+        <ListItem
+          // dummy strings temporary until load data into redux store during auto-login is not done
+          leftAvatar={{
+            rounded: true,
+            source: { uri: userImage || 'dummyUrl' },
+          }}
+          title={username || 'dummy name'}
+          titleStyle={{
+            color: Colors.navyBlue,
+            fontSize: 20,
+          }}
+          style={{ marginTop: 35, marginBottom: 35 }}
+          chevron
+        />
+      </TouchableWithoutFeedback>
+      <TouchableWithoutFeedback
+        onPress={() => Alert.alert('"Open view screen with some settings"')}
+      >
+        <ListItem
+          title="Test Option 1"
+          titleStyle={{
+            color: Colors.navyBlue,
+          }}
+          chevron
+          style={{ marginBottom: 5 }}
+        />
+      </TouchableWithoutFeedback>
+      <TouchableWithoutFeedback
+        onPress={() => Alert.alert('"Open view screen with some settings"')}
+      >
+        <ListItem
+          title="Test Option 2"
+          titleStyle={{
+            color: Colors.navyBlue,
+          }}
+          style={{ marginBottom: 5 }}
+          chevron
+        />
+      </TouchableWithoutFeedback>
+      <TouchableWithoutFeedback
```

Comment:
Just for discussion:
Don't you think this element should be more like a button? I'm not sure.

Reply:
What you mean "like a button"? Touch effects or look?

Reply:
No, I meant being an actual `Button` component instead of `TouchableWithoutFeedback` :smile: But after I had a read, in React Native there's no big difference between using `Button` and using `Touchable*` for custom button. I am probably used to web, where you have `<button />` and `<div />` elements, where using button is better for UX or accessibility.
I will just close this thread.